src/main/java/com/fcgl/madrid/dev/SampleDataSet.java

```diff
@@ -0,0 +1,131 @@
+package com.fcgl.madrid.dev;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fcgl.madrid.search.dataModel.Category;
+import com.fcgl.madrid.search.dataModel.Merchant;
+import com.fcgl.madrid.search.dataModel.Product;
```

Comment:
Department, Employee and Organization don't exist. These need to be taken out of the imports. Same for EmployeeRepository